src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Every time you need to add code, tests must be added or modified. In this case since you added a new controller make sure you add a new file for tests for this controller. Follow the files in /src/test/java/codeu/controller/ as examples. Also run the tests and make sure the all pass.